### PR TITLE
ACMS-1932: Capture and send Acquia Sites data to telemetry.

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #ACMS-1932

**Proposed changes**
Capture and send Acquia Sites data to telemetry.

**Alternatives considered**
NA

**Testing steps**
Follow from ticket.

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: acquia_cms.profile [hunk 1]
```diff
@@ -131,20 +123,34 @@ function acquia_cms_install_tasks(): array {
...
+  $appUuid = Environment::getAhApplicationUuid();
+  $siteGroup = Environment::getAhGroup();
+  $env = Environment::getAhEnv();
+  $acsfStatus = Environment::isAcsfEnv();
+  $siteUri = end(explode('/', \Drupal::getContainer()->getParameter('site.path')));
+  $siteName = \Drupal::config('system.site')->get('name');
+  $version = \Drupal::service('extension.list.module')->getExtensionInfo('acquia_cms')['version'];
+  $telemetryData = [
+    'acquia_cms' => [
+      'application_uuid' => $appUuid,
```
Comment (apathak18) [inline acquia_cms.profile:138]: @rajeshreeputra can we do something like this?
'application_uuid' => Environment::getAhApplicationUuid(),
'application_name' => Environment::getAhGroup(),
'environment_name' => Environment::getAhEnv(),
'acsf_status' => Environment::isAcsfEnv(),

File: acquia_cms.profile [hunk 2]
```diff
@@ -131,20 +123,34 @@ function acquia_cms_install_tasks(): array {
...
+      'application_name' => $siteGroup,
+      'environment_name' => $env,
+      'acsf_status' => $acsfStatus,
+      'site_uri' => $siteUri,
+      'site_name' => $siteName,
```
Comment (apathak18) [inline acquia_cms.profile:143]: If you think above suggestion is good to go then do update this
'site_name' => \Drupal::config('system.site')->get('name'),

File: acquia_cms.profile [hunk 3]
```diff
@@ -131,20 +123,34 @@ function acquia_cms_install_tasks(): array {
...
+      'starter_kit_name' => 'acquia_cms_existing_site',
+      'starter_kit_ui' => FALSE,
+      'site_studio_status' => $cohesion_configured ? TRUE : FALSE,
+      'profile' => 'acquia_cms',
+      'install_time' => $telemetry->calculateTime('install_start_time', 'install_end_time'),
+      'rebuild_time' => $telemetry->calculateTime('rebuild_start_time', 'rebuild_end_time'),
+      'version' => $version,
```
Comment (apathak18) [inline acquia_cms.profile:150]: Same here:
'version' => \Drupal::service('extension.list.module')->getExtensionInfo('acquia_cms')['version'],

File: acquia_cms.profile [hunk 4]
```diff
@@ -131,20 +123,34 @@ function acquia_cms_install_tasks(): array {
...
+    'acquia_cms' => [
+      'application_uuid' => $appUuid,
+      'application_name' => $siteGroup,
+      'environment_name' => $env,
+      'acsf_status' => $acsfStatus,
+      'site_uri' => $siteUri,
+      'site_name' => $siteName,
+      'starter_kit_name' => 'acquia_cms_existing_site',
+      'starter_kit_ui' => FALSE,
+      'site_studio_status' => $cohesion_configured ? TRUE : FALSE,
```
Comment (apathak18) [inline acquia_cms.profile:146]: $cohesion_configured => this variable itself returning 1 or 0 value which is same as TRUE or FALSE which you trying to implement, I guess below line is enough.
'site_studio_status' => $cohesion_configured

File: acquia_cms.profile (hunk 1 above)
Comment (rajeshreeputra) [reply acquia_cms.profile]: done

File: acquia_cms.profile (hunk 4 above)
Comment (rajeshreeputra) [reply acquia_cms.profile]: done.

File: acquia_cms.profile [hunk 5]
```diff
@@ -131,20 +123,7 @@ function acquia_cms_install_tasks(): array {
...
-    ->set('spi.amplitude_api_key', 'e896d8a97a24013cee91e37a35bf7b0b')
-    ->save();
-  $telemetry_service->sendTelemetry('acquia_cms_installed', [
-    'Application UUID' => Environment::getAhApplicationUuid(),
-    'Site Environment' => Environment::getAhEnv(),
-    'Install Time' => $telemetry->calculateTime('install_start_time', 'install_end_time'),
-    'Rebuild Time' => $telemetry->calculateTime('rebuild_start_time', 'rebuild_end_time'),
-    'Site Studio Install Status' => $cohesion_configured ? 1 : 0,
-  ]);
+  \Drupal::state()->set('acquia_cms.install_time', $telemetry->calculateTime('install_start_time', 'install_end_time'));
```
Comment (deepakmishra2) [inline acquia_cms.profile:126]: @rajeshreeputra so this mean we will not be sending data for 1.5.x ?

Comment (rajeshreeputra) [reply acquia_cms.profile:126]: We are sending telemetry data using the [acquia_cms_common/src/EventSubscriber/KernelTerminate/AcquiaCmsTelemetry.php](https://github.com/acquia/acquia_cms/blob/develop/modules/acquia_cms_common/src/EventSubscriber/KernelTerminate/AcquiaCmsTelemetry.php) event subscriber of Acquia CMS Common module.

**Review by apathak18 — APPROVED** [review]

**Review by deepakmishra2 — APPROVED** [review]

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**